db/schema.rb

```diff
@@ -10,30 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161129155123) do
+ActiveRecord::Schema.define(version: 20161202124612) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
 
-  create_table "taxpayers", force: :cascade do |t|
-    t.string   "email",                  default: "", null: false
-    t.string   "encrypted_password",     default: "", null: false
-    t.string   "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.integer  "sign_in_count",          default: 0,  null: false
-    t.datetime "current_sign_in_at"
-    t.datetime "last_sign_in_at"
-    t.inet     "current_sign_in_ip"
-    t.inet     "last_sign_in_ip"
-    t.datetime "created_at",                          null: false
-    t.datetime "updated_at",                          null: false
-    t.string   "tribunal_case_id"
-    t.index ["email"], name: "index_taxpayers_on_email", unique: true, using: :btree
-    t.index ["reset_password_token"], name: "index_taxpayers_on_reset_password_token", unique: true, using: :btree
-  end
-
```

Comment:
I accidentally checked this in a while back – it's good that this is coming out!